docs/source/config_file.rst

```diff
@@ -3,9 +3,12 @@
 The mypy configuration file
 ===========================
 
-Mypy supports reading configuration settings from a file.  By default
-it uses the file ``mypy.ini`` with a fallback to ``.mypy.ini``, then ``setup.cfg`` in
-the current directory, then ``$XDG_CONFIG_HOME/mypy/config``, then
+Mypy supports reading configuration settings from a file. 
+The configuration file may be generated or customized by users.
```

Comment:
This seems pretty obvious and doesn't seem worth spelling out -- this is how config files generally work.